src/reviews/api/dtos/CreateReviewDTO.ts

```diff
@@ -0,0 +1,17 @@
+export class CreateReviewDTO {
```

Comment:
I think DTO can just be an interface because we don't need any method here

Reply:
Yes, better to change it to interface only as it only need the shape.